demo/napari_gui/napari_.py

```diff
@@ -0,0 +1,26 @@
+# load data
+from skimage.io import imread
+image = imread('https://samples.fiji.sc/blobs.png')
+
+# start up napari
+import napari
+import pyclesperanto_prototype as cle
+with napari.gui_qt():
+    viewer = napari.Viewer()
+    viewer.add_image(image, name='blobs')
+
+    # push image to GPU
+    input = cle.push_zyx(image)
+
+    # process the mage
+    sigma = 3
+    blurred = cle.gaussian_blur(input, sigma_x=sigma, sigma_y=sigma)
```

Comment:
`sigma_x` makes me cringe. What do you think about refining the API here to allow vector sigma corresponding to the array axes?